src/main/scala/Chisel/Driver.scala

```diff
@@ -76,7 +76,7 @@ trait BackendCompilationUtilities {
         s"""-Wno-undefined-bool-conversion -O2 -DTOP_TYPE=V$dutFile -include V$dutFile.h""",
         "-Mdir", dir.toString,
         "--exe", cppHarness.toString)
-    System.out.println(s"${command.mkString(" ")}")
+    System.out.println(s"${command.mkString(" ")}") // scalastyle:ignore regex
```

Comment:
Just make this println(command.mkString(" ")).
The string interpolation is extranneous